jsapp/js/account/plans/plan.component.tsx

```diff
@@ -115,8 +115,9 @@ export default function Plan(props: PlanProps) {
   >([]);
   const [products, loadProducts, productsStatus] = useContext(ProductsContext);
   useRefreshApiFetcher(loadProducts, productsStatus);
-  const [organization, loadOrg, orgStatus] = useContext(OrganizationContext);
-  useRefreshApiFetcher(loadOrg, orgStatus);
+  const {
+    data: organizationData,
```

Comment:
Suggestion for a general naming convention (in the rest of this PR and the future): When naming component-level variables related to organizations, let's abbreviate to `org`. So `useOrganizationQuery`, for example, would stay as is, but `organizationData` would become `orgData`.

We're going to be adding a lot of org-related code in the near future, so I figure we should figure out these kinds of things now. Let me know what you think.

Reply:
I like doing something like `orgQuery = useOrganizationQuery();` and get the separate data/status/etc. through `orgQuery.data/status/etc.`